automation/clients/airtable.py

```diff
@@ -132,6 +133,25 @@ def validate_status(cls, v):
         return v
 
 
+############
```

Comment:
Yeah, the upside of split config is when you make changes to a function and its expected settings, it's localized to one module (plus you have to update the env files). The upside of centralized config is you can see the expected config in one module. I was leaning toward split config since we still have examples of the whole config in the env files (come to think of it we should have a test that the env files are all valid), but don't feel strongly.